be/src/common/config.h

```diff
@@ -379,6 +379,10 @@ CONF_mInt64(size_tiered_level_multiple, "5");
 CONF_mInt64(size_tiered_level_multiple_dupkey, "10");
 CONF_mInt64(size_tiered_level_num, "7");
 
+// disable compaction will take affect when tablet size is larger than this value
```

Comment:
skip the compaction against the tablets whose size is larger than this value.

Reply:
It is not that all tablets larger than this value are not compacted, but when disable compaction is turned on.

Reply:
that is exactly the problem. the explanation of this parameter is not very straightforward. I misunderstood it even by looking the code. How to expect end user to understand this parameter correctly. orz